web/src/components/post-onboarding/PostOnboardingRadioQuestion.tsx

```diff
@@ -1,17 +1,19 @@
 import { Content } from "@/components/Content";
 import { fetchPostOnboarding } from "@/lib/async-content-fetchers/fetchPostOnboarding";
 import { useUserData } from "@/lib/data-hooks/useUserData";
+import { postOnboardingCheckboxes } from "@/lib/domain-logic/postOnboardingCheckboxes";
 import { PostOnboarding } from "@/lib/types/types";
 import { ProfileData } from "@businessnjgovnavigator/shared/";
 import { FormControl, FormControlLabel, Radio, RadioGroup } from "@mui/material";
-import React, { ReactElement, useEffect, useState } from "react";
+import React, { ReactElement, useEffect, useMemo, useState } from "react";
 
 interface Props {
-  id: string;
+  postOnboardingQuestionId: string;
```

Comment:
```
postOnboardingQuestionId: keyof typeof postOnboardingCheckboxes;
```
[dust] If you wanted to be super specific about what kind of string we allow, we could limit this to just the keys of `postOnboardingCheckboxes`. But that is a little verbose and clunky to read. The juice might not be worth the squeeze.